zulipterminal/helper.py

```diff
@@ -421,7 +421,8 @@ def classify_unread_counts(model: Any) -> UnreadCounts:
             unread_counts['streams'][stream_id] = count
         else:
             unread_counts['streams'][stream_id] += count
-        unread_counts['all_msg'] += count
+        if stream_id not in model.muted_streams:
+            unread_counts['all_msg'] += count
```

Comment:
I didn't use `model.is_muted_stream()` because I can't figure out a way to mock it during testing, without rewriting the whole test.

Reply:
As per another comment, is this a bugfix for muted streams?